app/views/idv/in_person/state_id.html.erb

```diff
@@ -40,18 +40,33 @@
   </div>
 
   <div class="margin-bottom-4">
-    <%= render ValidatedFieldComponent.new(
-          name: :dob,
-          as: :date,
-          form: f,
-          hint: t('in_person_proofing.form.state_id.dob_hint'),
-          html5: true,
-          input_html: { value: pii[:dob] },
-          label: t('in_person_proofing.form.state_id.dob'),
-          label_html: { class: 'usa-label' },
-          maxlength: 255,
-          required: true,
-        ) %>
+  <%= render MemorableDateComponent.new(
+        content_tag: 'memorable-date',
+        name: :dob,
+        day: parsed_dob&.day,
+        month: parsed_dob&.month,
+        year: parsed_dob&.year,
+        required: true,
+        min: Time.zone.today - 200.years,
+        max: Time.zone.today,
+        hint: t('in_person_proofing.form.state_id.dob_hint'),
+        label: t('in_person_proofing.form.state_id.dob'),
+        form: f,
+        error_messages: {
+          missing_month_day_year: t('in_person_proofing.form.state_id.memorable_date.errors.date_of_birth.missing_month_day_year'),
+          range_overflow: t('in_person_proofing.form.state_id.memorable_date.errors.date_of_birth.range_overflow'),
+        },
+        range_errors: [
+          {
+            max: Time.zone.today - 13.years,
+            message: t(
+              'in_person_proofing.form.state_id.memorable_date.errors.date_of_birth.range_min_age',
+              app_name: APP_NAME,
+            ),
+          },
+        ],
```

Comment:
Wondering if this should be a hash instead of an array? Would that be easier to manage in the component implementation as a simple hash merge, vs. [what we have currently](https://github.com/18F/identity-idp/blob/ccf007332e483ef121561700c5c07da4a79df203/app/components/memorable_date_component.rb#L65-L72)? Is it possible to have multiple `max` or `min` errors?

```suggestion
        max: Time.zone.today - 13.years,
        range_errors: {
          max: t(
            'in_person_proofing.form.state_id.memorable_date.errors.date_of_birth.range_min_age',
            app_name: APP_NAME,
          ),
        },
```

Reply:
This was deliberately designed to allow multiple ranges that may overlap. The first invalid range in the list shows its validation message.

Reply:
I don't think that making this a hash would improve this implementation.